src/plone/restapi/tests/http-examples/vocabularies.resp

```diff
@@ -158,10 +162,6 @@ Content-Type: application/json
         "@id": "http://localhost:55001/plone/@vocabularies/plone.app.event.SynchronizationStrategies",
         "title": "plone.app.event.SynchronizationStrategies"
     },
-    {
-        "@id": "http://localhost:55001/plone/@vocabularies/plone.app.vocabularies.Catalog",
-        "title": "plone.app.vocabularies.Catalog"
-    },
```

Comment:
I think this file changed and is breaking the build because you ran the tests with some Python other than 3.9, which is what the CI build currently uses. (We should probably update that to 3.11)

Reply:
Thanks for the hint. Fixed it.